### PR TITLE
Fix fatal when viewing the customizer.

### PR DESCRIPTION
**Pull request by juanfra**
Fixing fatal.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
 # Theme compatibility for Hello Elementor.
 
 Add compatibility for Hello Elementor theme.
+
+### 1.0 — 2021-03-15
```
Comment (sc0ttkclark) [inline README.md]:
```suggestion
### 1.1 — 2021-03-15
```

File: index.php [hunk 2]
```diff
@@ -56,6 +56,10 @@ public function tec_tickets_ar_page_hello_elementor( $posts ) {
 			return $posts;
 		}
 
+		if ( is_admin() || is_customize_preview() ) {
```
Comment (sc0ttkclark) [inline index.php:60]: Maybe add comment as to what this prevents (fatal)

**Review by sc0ttkclark — APPROVED** [review]
Minor fix but looks good

File: README.md (hunk 1 above)
Comment (juanfra) [reply README.md]: Oh snap!

**Review by sc0ttkclark — APPROVED** [review]

**Pull request merged**